secure/key-manager/src/tests.rs

```diff
@@ -98,6 +98,20 @@ impl TestLibraInterface {
 }
 
 impl LibraInterface for TestLibraInterface {
+    fn last_reconfiguration(&self) -> Result<u64, Error> {
```

Comment:
why it's `last_reconfiguration`, it's just the latest timestamp?

Reply:
fixed, I intended to expose both, but brain fart...